swagger/v1/swagger.yaml

```diff
@@ -0,0 +1,413 @@
+---
```

Comment:
Isn't this file supposed to be auto-generated, does it need to be committed to git?

Reply:
both?

It could be committed -  which would make it a great static resource.
However, the tests currently generate volatile test data which means the examples would change every time the test is run.

I think I'll git ignore it, but I wanted you to see the output.